coops-ui/src/components/dialogs/WarningPopover.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-menu v-if="dialogObj != null"
+    :close-on-content-click="false"
+    :nudge-width="200"
+    offset-y
+    max-width="18rem"
+  >
+    <template v-slot:activator="{ on }">
+      <span class="warning-text" v-on="on">
+        <v-icon class="warning-icon">mdi-information-outline</v-icon>{{ dialogObj.title }}
+      </span>
+    </template>
+    <v-card>
+      <v-list-item>
+        <v-list-item-content>
+         {{ dialogObj.msg }}
+        </v-list-item-content>
+      </v-list-item>
+    </v-card>
+  </v-menu>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import { AlertMessage } from '../../interfaces'
```

Comment:
This would be better referenced as '@/interfaces'.

(@ = project-folder/src)

Reply:
sorry, vscode auto-inserted this